ui/src/app/pages/login/account-creation/account-creation-institution.spec.tsx

```diff
@@ -26,8 +30,8 @@ function component(): ReactWrapper {
   return mount(<AccountCreationInstitution {...props}/>);
 }
 
-function getInstance(wrapper: AnyWrapper): AccountCreationInstitution {
-  return wrapper.find(AccountCreationInstitution).instance() as AccountCreationInstitution;
+function getInstance(wrapper: AnyWrapper): AccountCreationInstitutionClass {
```

Comment:
`AccountCreationInstitution` is now a function component bc I used the `withStackdriverErrorReporterContext` HOC to wrap the component with the `Consumer`. so I'm exporting the wrapped class for testing because getting the class instance really does make the testing a lot easier.